app/pages/lab/workflows-container.jsx

```diff
@@ -50,6 +53,34 @@ export default class WorkflowsContainer extends React.Component {
     }
   }
 
+  handleSetStatsCompletenessType(e, page, workflow) {
+    workflow.update({ 'configuration.stats_completeness_type': e.target.value }).save()
+      .catch(error => console.log(error))
+      .then(() => this.getWorkflowList(page));
+  }
+
+  handleWorkflowStatusChange(e, page, workflow) {
+    const projectConfiguration = (this.props.configuration && this.props.configuration.default_workflow) ?
```

Comment:
This is storing the default workflow as `projectConfiguration`. I think @srallen was asking you to rename the `configuration` prop to `projectConfiguration` in order to distinguish it from the workflow config.